README.md

```diff
@@ -154,6 +154,43 @@ vim.keymap.set("n", "glP", gitlab.publish_all_drafts)
 vim.keymap.set("n", "glD", gitlab.toggle_draft_mode)
 ```
 
+### Or if you are using [which-key](https://github.com/folke/which-key.nvim), here's what you can use:
+
+```lua
+local wk = require("which-key")
+local gitlab = require("gitlab")
+
+wk.register({
```

Comment:
Hi, @chardskarth, please have a look at recent which-key's documentation. `wk.register` is deprecated ("Use `require("which-key").add()` instead.")